wicket-native-websocket/wicket-native-websocket-core/src/main/java/org/apache/wicket/protocol/ws/api/WebSocketResponse.java

```diff
@@ -47,9 +47,22 @@
 
 	private boolean isRedirect = false;
 
+	private final boolean asynchronous;
+
+	private final long timeout;
+
 	public WebSocketResponse(final IWebSocketConnection conn)
 	{
 		this.connection = conn;
```

Comment:
this can be replaced with `this(conn, false, -1)`

Reply:
Done